src/lib/classes/engine/Validation.ts

```diff
@@ -0,0 +1,24 @@
+const regexTest: RegExp = new RegExp(
+	"^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?).(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?).(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?).(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$", // Will test for any valid IP address
```

Comment:
Why not use [`URL.canParse`](https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static) for this? That would also allow you to support urls and things like `localhost`. I see no reason not to support that.

For the input in the DOM, you can just say `<input type="url">` and you're done.

Reply:
still here

Reply:
Jup, URL.canParse er ikke fuldt implementeret i Oracle hvilket er grunden til at det ikke blev ændret

Reply:
Jup, men I skal stadig acceptere hvis brugeren skriver en URL adresse ind, så I bliver nødt til at gøre et eller andet. Der er mange måder at løse dette på, en af dem er at læse sig frem til at Opera begynder at understøtte det i Januar 2024, så det har nok været understøttet rigeligt længe når UI'et begynder at blive brugt om ~2 år.